support-frontend/conf/routes

```diff
@@ -64,6 +64,15 @@ GET  /$countryGroupId<(uk|us|au|eu|int|nz|ca)>/checkout            controllers.A
 GET  /$countryGroupId<(uk|us|au|eu|int|nz|ca)>/one-time-checkout   controllers.Application.productCheckoutRouter(countryGroupId: String)
 GET  /$countryGroupId<(uk|us|au|eu|int|nz|ca)>/thank-you           controllers.Application.productCheckoutRouter(countryGroupId: String)
 
+GET  /$countryGroupId<(uk)>/guardian-light                         controllers.Application.productCheckoutRouter(countryGroupId: String)
+GET /us/guardian-light                                             controllers.Application.contributeGeoRedirect(campaignCode = "")
```

Comment:
Could you combine the redirect countries into one line?

Reply:
We tried, there appears to be no examples of this and only of them broken out into regions. We tried `us|eu...` variants and regex but no luck.  Can revisit if we find out.